ext/B/t/concise.t

```diff
@@ -131,9 +127,6 @@ sub render {
 
 SKIP: {
     # tests output to GLOB, using perlio feature directly
-    skip "no perlio on this build", 127
-	unless $Config::Config{useperlio};
-    
```

Comment:
As above: If we're removing the `skip()` function, shouldn't we also remove the `SKIP: { }` block (and de-intent the remaining code)?

Reply:
As above. (done)